src/main/java/Airport.java

```diff
@@ -25,108 +27,63 @@
     }
 
 
-    public boolean isPassengerPlane(Plane plane) {
-        boolean isPasenger = false;
-        if (plane instanceof PassengerPlane) {
-            isPasenger = true;
-        } else {
-            LOGGER.info("Plane's model is not passenger - will be skipped");
-        }
-        return isPasenger;
-    }
-
-
-    public List<MilitaryPlane> getMilitaryPlanes() {
-        List<MilitaryPlane> militaryPlanes = new ArrayList<MilitaryPlane>();
-        planes.forEach(plane ->
-        {
-            if (isMilitaryPlane(plane)) {
-                militaryPlanes.add((MilitaryPlane) plane);
-            } else {
-                try {
-                    throw new NonMilitaryPlainException("Plane is not military");
-                } catch (NonMilitaryPlainException e) {
-                    LOGGER.info(e.getMessage());
-                }
-            }
-        });
-        return militaryPlanes;
+    private boolean isPassengerPlane(Plane plane) {
+        return (plane instanceof PassengerPlane) ? true : false;
     }
 
 
     protected boolean isMilitaryPlane(Plane plane) {
-        boolean isMilitaryPlane = false;
-        if (plane instanceof MilitaryPlane) {
-            isMilitaryPlane = true;
-        } else {
-            LOGGER.info("Plane is not military");
-        }
-        return isMilitaryPlane;
+
+        return (plane instanceof MilitaryPlane) ? true : false;
     }
 
     public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() {
-        List<PassengerPlane> passengerPlanes = getPassengerPlane();
-        PassengerPlane planeWithMaxCapacity = passengerPlanes.get(0);
-        for (int i = 0; i < passengerPlanes.size(); i++) {
-            planeWithMaxCapacity = comparePlanesByCapacity(planeWithMaxCapacity, planeWithMaxCapacity);
-        }
-       return  planeWithMaxCapacity;
+        List<PassengerPlane> passengerPlanes = getPassengerPlanes();
+        AtomicReference<PassengerPlane> planeWithMaxCapacity = new AtomicReference<>(passengerPlanes.get(0));
+        passengerPlanes.forEach(plane -> {
+            planeWithMaxCapacity.set(comparePlanesByCapacity(planeWithMaxCapacity.get(), plane));
+        });
```

Comment:
here's a better solution. with java 8 comparator:

planeWithMaxCapacity = Collections.max(passengerPlanes, Comparator.comparing(Plane;:getCapacity));

and we don't need comparePlanesByCapacity method in this case